---
name: 2 - Bug report
about: Something isn't working as expected.
title: ''
labels: bug
assignees: ''

---

### Required attestation
- [ ] I am using a virtual environment that matches the book specifications **exactly**.
- [ ] I confirm that my fork is up to date with the latest changes in this repository.
- [ ] I have checked that this issue has not already been reported or resolved [here](https://github.com/stefmolin/Hands-On-Data-Analysis-with-Pandas-2nd-edition/issues?q=is%3Aissue).
- [ ] I have confirmed that my results match those obtained when running the code using [this](https://mybinder.org/v2/gh/stefmolin/binder-environments/2nd_edition?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fstefmolin%252FHands-On-Data-Analysis-with-Pandas-2nd-edition%26urlpath%3Dlab%252Ftree%252FHands-On-Data-Analysis-with-Pandas-2nd-edition%252F%26branch%3Dmaster) Binder environment.

---

### Background information
##### 1. Which OS are you using?
TODO: Provide your OS here &ndash; make sure to differentiate between Intel and M1 chip Macs.

##### 2. Which Python version are you using?
TODO: Provide your Python version here.

##### 3. Are you using `conda` or `venv`?
TODO: Indicate whether you are using `conda` or `venv`.

##### 4. Package versions
<details>
<summary>versions installed</summary>

```
TODO: Paste the result of running `pip freeze` or `conda list`
```

</details>

##### 5. Run the `ch_01/checking_your_setup.ipynb` notebook
Screenshot after running the `ch_01/checking_your_setup.ipynb` notebook:

TODO: paste your screenshot here

---

### Commands run and their outputs
Please provide **all** of the commands you ran as well as the traceback:

<details>

```
TODO: paste commands and any traceback here
```


</details>

### Screenshots
Optionally, include any screenshots that will help diagnose the issue.
